pasqal_cloud/client.py

```diff
@@ -44,33 +44,41 @@ class EmptyFilter:
 
 
 class Client:
-    authenticator: AuthBase
+    authenticator: AuthBase | None
 
     def __init__(
         self,
-        project_id: str,
+        project_id: str | None = None,
         username: Optional[str] = None,
```

Comment:
authentificator: Union[str, None]

project_id: Optional[str] = None,

what about this to be consistent with the others ? The pipe was added in python 3.10 however we need to make sure pasqal-cloud is compatible with python 3.8

Reply:
In a previous PR, @oliver-gordon  suggested using the new way to declare optional parameters (`type | None = None`). I think, this is the right approach, especially since it’s the new standard and our checks/tests are passing

Reply:
Did they patch python 3.8 to make this work? Because in the past this would raise an error